app/models/contact.js

```diff
@@ -0,0 +1,15 @@
+import Ember from 'ember';
+import DS from 'ember-data';
+
+export default DS.Model.extend({
+  emailAddress: DS.attr('string'),
+  message: DS.attr('string'),
+  // validations
+  validEmail: Ember.computed.match('emailAddress', /^.+@.+\..+$/),
+  validMessage: Ember.computed('message', function () {
+    const msg = this.get('message');
+    return (msg && msg.trim().length > 5);
+  }),
+  isValid: Ember.computed.and('validEmail', 'validMessage'),
+  isNotValid: Ember.computed.not('isValid'),
```

Comment:
@SentientBicycle and @jasonly I probably don't need 'isValid' attribute but keep 'isNotValid' and use a computed function to return `! this.get('validEmail') && this.get('validMessage');`